Scheduler/observing_blocks_maintel/BLOCK-T74/BLOCK-T74_1.json

```diff
@@ -0,0 +1,412 @@
+{
+    "name": "BLOCK-T74_1",
+    "id": "e7afd17e-a473-4890-a2e0-510582b318ad",
+    "program": "BLOCK-T74_1",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "track_azel": {
+                    "az": -180,
+                    "el": 80
+                }
+            }
+        },
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "filter": "r_03",
```

Comment:
I'm torn on whether we should just omit the filter here (in which case take_triplet_comcam will use the current filter) or should we make the filter configurable, or leave it hard-coded as r-band.  Maybe something to bring up and discuss more generally at a future meeting.

Reply:
I thought the same as I was hard-coding it in. Having a user-provided filter would be the safest (in case we get the opportunity to execute it on a night when the r filter is not installed.